code/controllers/subsystems/ticker.dm

```diff
@@ -438,6 +438,23 @@ var/datum/controller/subsystem/ticker/SSticker
 
 	to_world("<B><span class='notice'>Welcome to the pre-game lobby!</span></B>")
 	to_world("Please, setup your character and select ready. Game will start in [pregame_timeleft] seconds.")
+
+	// Compute and, if available, print the ghost roles in the pre-round lobby. Begone, people who do not ready up to see what ghost roles will be available!
+	var/list/available_ghostroles = list()
+
+	for(var/s in SSghostroles.spawners)
+		var/datum/ghostspawner/G = SSghostroles.spawners[s]
+		if(G.enabled && !("Antagonist" in G.tags) && !(G.loc_type == GS_LOC_ATOM && !length(G.spawn_atoms)))
+			available_ghostroles |= list(G.name)
```

Comment:
```suggestion
			available_ghostroles |= G.name
```

This doesn't need to be a list.

Reply:
https://secure.byond.com/docs/ref/index.html#/operator/|=

"If A _and_ B are lists, any items in B that are not already in A are added to A."

I take it that it's expected that both sides are, for the operator?
Also, is there any reason not to have it as list?

Reply:
That line is just about special behaviour if you have two lists, if you have A (list) and B (non-list) it will insert B into A so long as A doesn't already contain B.

Defining a list takes up more resources than not defining a list. In this case the behaviour is the same.